network/endpoint_windows.go

```diff
@@ -381,6 +393,12 @@ func (nw *network) newEndpointImplHnsV2(cli apipaClient, epInfo *EndpointInfo) (
 		gateway = net.ParseIP(hnsResponse.Routes[0].NextHop)
 	}
 
+	nicName := epInfo.IfName
+	// infra nic nicname will look like eth0, but delegated/secondary nics will look like "vEthernet 2"
```

Comment:
NIT: infra NIC name on windows always look like eth0(or VEthernet 0); secondary interfaces name look like "vEthernet x"(x means 1-7, maximum up to 8 interfaces on one windows node)